controllers/flowcollector_controller_certificates_test.go

```diff
@@ -146,6 +146,13 @@ func flowCollectorCertificatesSpecs() {
 			DeploymentModel: flowslatest.DeploymentModelKafka,
 			Agent: flowslatest.FlowCollectorAgent{
 				Type: "eBPF",
+				EBPF: flowslatest.FlowCollectorEBPF{
+					Advanced: &flowslatest.AdvancedAgentConfig{
+						Env: map[string]string{
+							"DEDUPER_JUST_MARK": "true",
```

Comment:
isn't that the default ? why need to set it ?

Reply:
That's the default indeed !

However removing it seems to affect the reconcile loop and makes the test fail. As far as I remember it says the CR failed to update because it's not up to date (probably from status update code). It would require a dummy update in the tests to make it work.

Everything seems to work fine on a real cluster :smile_cat:

Reply:
yeah this is just test code